projects/1.20/assets/dungeons-delight/dungeons_delight_gourmet_battles/lang/zh_cn.json

```
@@ -0,0 +1,91 @@
+{
+  "advancements.crispyandsweet.descr": "这些腿尝起来怎么样？",
+  "advancements.crispyandsweet.title": "香脆甜美",
+  "advancements.dragon_delight.descr": "你烹饪了末地的守护者",
+  "advancements.dragon_delight.title": "龙之乐事",
+  "advancements.dungeon_apprentice.descr": "准备好开始地下城烹饪之旅了吗？",
+  "advancements.dungeon_apprentice.title": "地下城新手",
+  "advancements.flamingpudding.descr": "甜蜜的痛苦？火热的甜蜜？",
+  "advancements.flamingpudding.title": "火焰布丁",
+  "advancements.ghostlyhanbuger.descr": "快速准备好的恶魂肉！",
+  "advancements.ghostlyhanbuger.title": "恶魂汉堡",
+  "advancements.greenpizzeria.descr": "你做出了意大利奇迹！",
```

```suggestion
  "advancements.greenpizzeria.descr": "你做出了绿色的意大利奇迹！",
```
漏译`green`
奇迹是否是奇观（文明系列）有待讨论